app/assets/stylesheets/partials/_dashboard.scss

```diff
@@ -23,6 +23,24 @@ Dashboard UI
   background: #fff;
 }
 
+.empty {
```

Comment:
Want to use a name like "coming-soon" or "preview"? I can see us using "empty" to signify a card that doesn't have any data to show yet (like at a facility that hasn't registered patients yet), and I don't think we'd want the green highlighting for those.

Reply:
Good idea. Fixing